src/azure-cli/azure/cli/command_modules/backup/_help.py

```diff
@@ -502,6 +502,16 @@
     text: az backup vault show --name MyRecoveryServicesVault --resource-group MyResourceGroup
     crafted: true
 """
+
+helps['backup vault soft-deleted-containers'] = """
+type: command
+short-summary: List soft-deleted containers within a particular Recovery Services vault.
+examples:
+  - name: List soft deleted containers
+    text: az backup vault soft-deleted-containers --name MyRecoveryServicesVault --resource-group MyResourceGroup --backup-management-type AzureIaasVM
+    crafted: true
+"""
```

Comment:
In CLI, all command names should contain a verb ([Command Naming and Behavior](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#command-naming-and-behavior)).
Since this command is to list soft deleted containers, it is suggested to rename the command with a `list` word.

Reply:
Renamed to `list-soft-deleted-containers`. Also added new commands/parameters in history notes.